shared/actions/chat/manage-thread.js

```diff
@@ -41,7 +41,10 @@ function* _startConversation(action: ChatGen.StartConversationPayload): Saga.Sag
 
   if (forceImmediate && existing) {
     const newID = yield Saga.call(Shared.startNewConversation, existing.get('conversationIDKey'))
-    yield Saga.put(ChatGen.createSelectConversation({conversationIDKey: newID}))
+    if (!newID[0]) {
+      throw new Error('Unable to get new conversation ID')
```

Comment:
what exactly was the bug here? won't throwing an error still bring up a black bar?

Reply:
`Shared.startNewConversation` explicitly returns an array `[conversationIDKey, tlfName]` or `[null, null]` if there's an error in `RPCChatTypes.localNewConversationLocalRpcPromise`. 

The old code was treating the result of `startNewConversation` as a single value, which always threw the typeerror. This catches the rpc error instead.